doc/cephadm/services/monitoring.rst

```diff
@@ -171,9 +171,15 @@ configuration files for monitoring services.
 
 Internally, cephadm already uses `Jinja2
 <https://jinja.palletsprojects.com/en/2.11.x/>`_ templates to generate the
-configuration files for all monitoring components. To be able to customize the
-configuration of Prometheus, Grafana or the Alertmanager it is possible to store
-a Jinja2 template for each service that will be used for configuration
+configuration files for all monitoring components. Starting from version x.x.x, 
+cephadm uses Prometheus http service discovery support `http_sd_config
+<https://prometheus.io/docs/prometheus/2.28/configuration/configuration/#http_sd_config>`
+in order to get the currently configured targets from Ceph. Internally, `ceph-mgr`
+provides a service discovery endpoint at `<https://<mgr-ip>:8765/sd/` which used
+by Prometheus to get the needed targets.
+
+To be able to customize the configuration of Prometheus, Grafana or the Alertmanager
```

Comment:
Suggest:
The configuration of Prometheus, Grafana, or Alertmanager may be customized by storing a Jinja2 template for each service.

Reply:
@anthonyeleven fixed :+1: